dojo/unittests/test_api_v1.py

```diff
@@ -72,9 +72,22 @@ def test_create_product(self):
         p = Product.objects.get(id=obj_id)
         self.assertEqual('Fantastic Product', p.name)
 
-    @skip("TODO: Test for product updating")
+    # @skip("TODO: Test for product updating")
     def test_update_product(self):
-        pass
+        new_p_data = {
```

Comment:
Not sure what this does. Need another reviewer to check this out.

Reply:
Oh.. i did that earlier. while I was wrapping my ahead around the test in dojo/unittest folder.

It is a unittest to update product using api_v1.